website/docs/faq.md

```diff
@@ -160,6 +160,53 @@ out-of-bounds. If you would rather raise an exception than handle `nil`, use the
 [0, 1, 2].fetch(3) # IndexError: index 3 outside of array bounds
 ```
 
+## The types for `Array#sample` are weird
```

Comment:
Can you generalize this heading to say something about how overloads + keyword arguments are unsupported, moving Array#sample to a specific example of the general behavior, and avoid using the term “weird” instead of something more precise?

Reply:
Can do. Apart from sample, are there other common examples of this worth calling out?

Reply:
https://github.com/sorbet/sorbet/blob/b0537331e412423cbc494d74bd9d39144d39b837/rbi/core/kernel.rbi#L761

Also #37 mentions Dir.glob (#37 is the more general issue to link to)

Reply:
https://github.com/sorbet/sorbet/pull/4214

Another one

Reply:
Thanks!! I generalised it based on this feedback, and added some more examples.